Sources/XCTest/Private/PrintObserver.swift

```diff
@@ -59,7 +59,9 @@ internal class PrintObserver: XCTestObservation {
 
     fileprivate func printAndFlush(_ message: String) {
         print(message)
+        #if !os(Android)
         fflush(stdout)
+        #endif
```

Comment:
I'll defer to @briancroom on this one. Does Android really not provide a `stdout` file handle? Is this a shortcoming in the Swift Glibc overlay when built for Android? If so, I guess I'm fine with this change.

Reply:
Yeah, this is interesting! I dug a little bit and found that `stdout` isn't being build for Android currently: https://github.com/apple/swift/blob/48308411393e730cf3cb21d353a9be31045c47e4/stdlib/public/Platform/Platform.swift#L98

And here's a bug about a Swift test failure caused by the same issue (guess who filed it? 🙃): https://bugs.swift.org/browse/SR-1835

It looks like these could be brought in only by building against Android API level 23, which may be an acceptable change at some point in the future.